client/Models/EditorModel.ts

```diff
@@ -0,0 +1,17 @@
+import { atom } from 'jotai';
+
+// Default Editor Configurations
+export const editorSettingsAtom = atom({
```

Comment:
Use `atomWithStorage` to have the settings automatically save.

```suggestion
export const editorSettingsAtom = atomWithStorage('editor-storage', {
```